titles/install-rhdh-aks/master.adoc

```diff
@@ -6,4 +6,15 @@ include::artifacts/attributes.adoc[]
 :imagesdir: images
 
 // aks deployment
-include::assemblies/assembly-install-rhdh-aks.adoc[leveloffset=+1]
+//include::assemblies/assembly-install-rhdh-aks.adoc[leveloffset=+1] //RHIDP-4165: multi-chapter format
```

Comment:
Consider removing the include statement lines and deleting the assemblies that are not used anymore.

Reply:
I've removed the assembly files from the repo. I think we should keep the include as a reference, just in case any questions arise about this change.